geode-assembly/geode-assembly-test/src/main/java/org/apache/geode/rest/internal/web/controllers/GetAllEntries.java

```diff
@@ -29,6 +29,7 @@
 
 public class GetAllEntries implements Function {
 
+  @SuppressWarnings("unchecked")
```

Comment:
Is it possible to add correct generic types here so we don't need to suppress anything?

Reply:
I have removed this change. I don't know how it snuck in. It caused a bunch of merge conflicts. The build is fine without this change. If someone wants to clean this test code up I think it should be on another PR.